lib/Application/Complete.php

```diff
@@ -160,11 +160,15 @@ private function populateSuggestions(SymbolContext $symbolInformation, Type $typ
             return $symbolInformation->withIssue(sprintf('Could not find class "%s"', (string) $type));
         }
 
+        $needsPublicAccess = !in_array($symbolInformation->symbol()->name(), ['this', 'self']);
```

Comment:
Rename to `$publicOnly`? also `false === in_array` rather than `!`

Reply:
`in_array` only returns booleans though :P

Reply:
I'd be more worried about not setting its 3rd argument to `true`

Reply:
I agree with greg0ire.

@dantleech if you want or it's the coding style of this project I will change it accordingly.

Reply:
> in_array only returns booleans though :P

Strict comparators and boolean keywords are used mostly in the code base - I find it easier to read, but its subjective. Main thing is consistency.

Reply:
> if you want or it's the coding style of this project I will change it accordingly.

Nah, its fine. Not sure what my algorithm is for the two styles anyway.